src/test.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 		port: 3000,
 		corsOptions: {
 			maxAge: 86400,
-			methods: 'HEAD,PUT,PATCH,POST,DELETE'
+			methods: 'HEAD,PUT,PATCH,POST,DELETE',
```

Comment:
no GET?

Reply:
Yea I dunno, not sure why it is not in here, I just ran prettier on this so kind of glad prettier added that trailing comma

Reply:
was prob me...